File: src/main/java/macro303/neptunes/NeptunesApp.java
package macro303.neptunes;

import javafx.application.Application;
import javafx.fxml.FXMLLoader;
import javafx.scene.Parent;
import javafx.scene.Scene;
import javafx.stage.Stage;

public class NeptunesApp extends Application {

	public static void main(String[] args) {
		launch(args);
	}

	@Override
	public void start(Stage primaryStage) throws Exception {
		Parent root = FXMLLoader.load(getClass().getResource("NeptunesView.fxml"));

		Scene scene = new Scene(root);
		primaryStage.setTitle("Neptune's Pride");
		primaryStage.setScene(scene);
		primaryStage.show();
	}
}

File: src/main/kotlin/macro303/neptunes/NeptunesApp.kt
package macro303.neptunes

import javafx.stage.Stage
import tornadofx.*

/**
 * Created by Macro303 on 2018-05-08.
 */
class NeptunesApp : App(primaryView = NeptunesView::class) {
	init {
//		importStylesheet(stylesheet = "/css/modena_dark.css")
//		importStylesheet(stylesheet = "/css/bootstrap3.css")
		importStylesheet(stylesheet = "/css/MaterialFX-v0.3.css")
//		importStylesheet(stylesheet = "/css/sdkfx.css")
	}

	override fun start(stage: Stage) {
		super.start(stage)
		stage.isMaximized = true
	}
}